uber/custom_tags.py

```diff
@@ -49,11 +57,15 @@ def event_dates():
 
 @JinjaEnv.jinja_export
 def hour_day_local(dt):
+    if not dt:
+        return ''
     return hour_day_format(dt)
 
 
 @JinjaEnv.jinja_filter
 def timestamp(dt):
+    if not dt:
```

Comment:
The fact that you had to add the same code block to five different functions indicates to me that these tags could probably use refactoring. Probably out of scope for this PR, but something to think about.